AspNetCore3ChildlifetimeScope/StartupApplicationA.cs

```diff
@@ -0,0 +1,33 @@
+using AspNetCore3ChildlifetimeScope.Dependencies;
+using Autofac;
+using Autofac.Extensions.DependencyInjection;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace AspNetCore3ChildlifetimeScope
+{
+    public class StartupApplicationA
+    {
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddControllers();
+        }
+
+        public void Configure(IApplicationBuilder app)
+        {
+            app.UseRouting();
+
+            app.UseEndpoints(builder => builder.MapControllers());
+        }
+
+        public void ConfigureContainer(AutofacChildLifetimeScopeConfigurationAdapter configurationAdapter)
+        {
+            configurationAdapter.Add(
```

Comment:
Some sort of comment here (and in the associated `StartupApplicationB` class) would help draw peoples' eyes to the "important part" of the demo.
```suggestion
            // Using the configuration adapter will cause
            // the registrations to occur in an application-specific
            // child lifetime scope.
            configurationAdapter.Add(
```

Reply:
Unclear how the suggestion feature here will work, if it will actually pull in the additional comment or if you'll have to copy/paste.